src/LearningObjects/Publishing/Bundler/Interactor.ts

```diff
@@ -163,19 +164,19 @@ function addFiles({
 /**
  * addChildren triggers the process of building a bundle structure for each child of a Learning Object.
  *
- * @param {LearningObject[]} children [Array of child Learning Objects]
+ * @param {HierarchicalLearningObject[]} children [Array of child Learning Objects]
  * @param {string}: prefix [File path prefix (ie. fileName: 'World.txt', prefix: 'Hello' = filePath: 'Hello/World.txt')]
  * @returns {Promise<BundleData[]>}
  */
 async function addChildren({
   children,
   prefix = '',
 }: {
-  children: LearningObject[];
+  children: HierarchicalLearningObject[];
   prefix: string;
 }): Promise<BundleData[]> {
   const childrenBundleData = await Promise.all(
-    children.map(child => {
+    children.map(async child => {
```

Comment:
Async not needed here
```suggestion
    children.map(child => {
```

Reply:
The `loadReleasedChildObjects` function returns a Promise so removing the async label seems to break it.